.eslintrc.json

```diff
@@ -0,0 +1,54 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "overrides": [
+        {
+            "files": [
+                "**/*.cy.js"
+            ],
+            "env": {
+                "cypress/globals": true
+            },
+            "plugins": [
+                "cypress"
+            ],
+            "extends": [
+                "plugin:cypress/recommended"
+            ],
+            "rules": {
+                "cypress/no-unnecessary-waiting": "off",
+                "no-unused-vars": "off"
+            }
+        },
+        {
+            "files": [
+                "**/*.test.js",
+                "**/*.mock.js"
+            ],
+            "env": {
+                "jest": true
+            },
+            "plugins": [
+                "jest"
+            ],
+            "extends": [
+                "plugin:jest/recommended"
+            ],
+            "rules": {
+                "jest/prefer-expect-assertions": "off"
+            }
+        }
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "rules": {
+        "no-undef": "off",
```

Comment:
Ideally you should avoid changing global rules and stick to the overrides instead. These rules are helpful for the normal app code.